resources/v1/clusters/requests.go

```diff
@@ -12,11 +12,11 @@ type ListOptsBuilder interface {
 
 // ListOpts contains parameters for filtering a List request
 type ListOpts struct {
-	Detail   bool   `q:"detail"`
-	Local    bool   `q:"local"`
-	Area     string `q:"area"`
-	Service  string `q:"service"`
-	Resource string `q:"resource"`
+	Detail   bool     `q:"detail"`
+	Local    bool     `q:"local"`
+	Area     []string `q:"area"`
+	Service  []string `q:"service"`
+	Resource []string `q:"resource"`
```

Comment:
If multiple entries are possible, the identifiers should be in plural: `Areas`, `Services`, `Resources`. (Same below on the other `ListOpts` and `GetOpts`.)